problems/euler/52/52.py

```diff
@@ -0,0 +1,39 @@
+"""
+Euler Problem: 52
```

Comment:
wait, the pull request title says "Add python solution for euler#53", but this is the solution for 52. Was that a mistake?